templates/module/Source/module/Private/Generated/Monitor/pluginfactory.cpp.tpl

```diff
@@ -46,7 +56,10 @@ DEFINE_LOG_CATEGORY(Log{{$mclass}});
 {{- if $.Features.olink }}
 TScriptInterface<I{{$class}}Interface> create{{$class}}OLink(UGameInstance* GameInstance, FSubsystemCollectionBase& Collection)
 {
-	UE_LOG(Log{{$mclass}}, Log, TEXT("create{{$iclass}}: Using OLink service backend"));
+	if (IsLogEnabled())
```

Comment:
There are a lot of calls to IsLogEnabled() - There could be a potential for coming up with a specific ApiGearLog function that calls IsLogEnabled and UE_LOG.

Reply:
Yes, thought about it but UE_LOG is macro and not a simple function.
So you need to forward those macro arguments to a function and then to the macro.
Also there a different settings for the logging categories, so you would need different functions to check the correct settings and not one global.

All those checks esp. including the settings are performance hits which are probably not necessary.
Thus, I am thinking about removing all those checks. Logging can be disabled within UE by simply choosing the logging categories.

Reply:
Makes sense!